app/views/content/explore-teaching-advisers/_ending.html.erb

```diff
@@ -1,44 +1,24 @@
 <div class="row inset">
-  <section class="col col-content-max">
-    <%= render TextBlockWithOneImageComponent.new(
-      heading: "Is teaching right for me?",
-      colour: "yellow",
-      image_path: "static/images/content/hero-images/0011.jpg",
-      image_alt: "Child pointing something out to a teacher in the classroom.",
-    ) %>
-  </section>
-
-  <section class="col col-720">
-    <h2 class="heading-s">Use your subject passion</h2>
-
-    <p>
-    Bring your subject to life for your pupils. Help them be as inspired by it as you are.
-    </p>
-
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">Is teaching right for me?</h2>
+  <h3>Use your subject passion</h3>
+  <p>Bring your subject to life for your pupils. Help them be as inspired by it as you are.</p>
```

Comment:
Not sure of value to users of this section, especially if main page aim is to get people to sign up to the advisers? Let's discuss tomorrow.